### ISSUE TITLE
Author info not displayed on featured content

### ISSUE THREAD

**Issue by davoraltman** [Automattic/themes#30]
## Steps to replicate

I am not sure if this should be submitted as a bug or as an enhancement request. When looking at the featured content for the Canard theme, I can only see a Category displayed beside the title. The author info (and tags and date) aren't displayed - http://cld.wthms.co/TN2MEy

## Expected

If this is an expected behavior and the aforementioned are only supposed to be displayed on single posts, then this can be an enhancement request! If not, this is a bug. Tagging with needs testing so that we can determine that. Thanks!

* User report: Came up during 1:1 session
* Follow-up ticket: 988233-zen
* User's site: https://yankeebrigade.com but I also replicated on my non-AT site https://customerhappiness.rs/

**Comment by sixhours** [comment]
I can confirm this would be an enhancement. The theme was not originally designed with these elements in the Featured Content area.

**Comment by alaczek** [comment]
I'm not 100% sold we should go forward with this enhancement. Featured Content is a special case, and I don't think Content Options settings are being applied to Featured Content in any of the other themes.

**Comment by kathrynwp** [comment]
Agree. With only one request in all the time this theme has existed, I don't think this is worth the time to implement. Since the user has custom CSS, an author can be added that way if they really wanted.
